guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,360 @@
+
+After the first use, users are typically on their own to use the application as it fits their needs, and everything should be set up appropriately. Features that are typically only relevant at later stages of the [usage life cycle]({{ '/guide/designing-products/usage-life-cycle/#first-use' | relative_url }}) may benefit from their own onboarding flows. Key is to present introductory information at the time it becomes relevant to the user.
+
+{% include image-gallery.html pages = page.images_test-txns %}
+
+## Receiving & scanning
+
+Since static addresses can now be used to pay a receiver several times, receiving flows are likely to be used less often, rarely even, since static addresses can be posted in public and others can simply get it from there. So whenever the receiver is sharing a payment request, users have the opportunity to associate valuable information to it – through labels and contacts; applications should facilitate this process..
+
+Some places where people can share static addresses (and add labels) to include:
+- Social media pages
+- Donate or fundraising websites
+- Exchanges
+- Email signatures
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   modalImage = "/assets/images/guide/how-it-works/silent-payments/adding-labels-to-static-address@2x.png"
+   width = 250
+   height = 522
+   alt-text = "An application screen showing labels being added to a static address."
+   caption = "."
+   layout = "float-right-desktop -background -shadow"
+%}
+
+Regularly [backing up wallet]({{ '/guide/daily-spending-wallet/backup-and-recovery/landing-page/' | relative_url }}) and user data in a secure manner is essential for bitcoin applications. For small amounts of funds, [encrypted cloud backups]({{ '/guide/daily-spending-wallet/backup-and-recovery/cloud-backup/' | relative_url }}) are a convenient choice with a good tradeoff between security and convenience.
+
+Frequent backups are important for lightning wallets, as channel states frequently change and need to be updated.
+
+</div>
+
+
+
+Receivers using mobile wallets, which may not be online 24x7 will encounter some delay (once they come online) as their wallet scans the blockchain and performs calculations to detect their payments. Applications should communicate this fact during setup/onboarding. While the scanning itself is taking place, applications should show progress data and estimated time to complete the scanning process.
+
+The time to detect a specific received payment can be minimized if the sender & receiver can communicate with each other. Some applications already allow users to detect whether an address belongs to them. This feature should, if possible, be extended to include BIP-352 functionality.
+
+## Backup
+
+Like lightning wallets, on-chain bitcoin wallets supporting full-featured BIP-352 wallets require file backups. These backup files may contain the following information:
```

Comment:
For backup and recovery, how about just link out to the [backup and recovery section](https://bitcoin.design/guide/daily-spending-wallet/backup-and-recovery/landing-page/), with a small note about labels...?

Just feels like we're repeating some things here, and the point about wallet birthday is not specific to silent payments, and could be tackled on that page as well.

Reply:
I have now linked to the user meta section on that page for a good explanation about the need to backup that in https://github.com/BitcoinDesign/Guide/pull/1109/commits/30efc0019bd71fdc20a72422ecb349b9f5d94e33

I am not able to find mention of wallet bitrthday in the content, though the [backup print template](
![](https://bitcoin.design/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/manual-backup-print-detail-full.png)
) does mention a creation date). Should I add content to that page mentioning birthday if something doesn't already exist?